### ISSUE TITLE
Color Contrast on Departments

### ISSUE THREAD

**Issue by david-iterators** [CityOfBoston/Iterators#249]
Brief Description
Color Contrast on Departments

Screengrab/videos (insert file names here)
See below

Tester (first name)
David

Link
Production URL:  https://boston.gov/departments/311/city-boston-government
Stage URL:  https://d8-stg.boston.gov/departments/311/city-boston-government

Environment
(Device > OS > Browser)
Computer > macOS > Safari

Type of Bug:  Functional, Accessibility or Content
Accessibility

Steps to reproduce
1)	Go to the URL and scroll to the City Council
a)	BUG:  The text for “Send an email” is blue on blue and not sufficient color contrast

Desired Result
•	Some text has insufficient color contrast

Actual Result
•	Insufficient Color Contrast
<img width="1526" alt="Color Contrast on Departments" src="https://user-images.githubusercontent.com/33845414/149252030-5e0e5b93-fb8d-428a-8415-42f415323a8b.png">

**Comment by subaha-cob** [comment]
@david-iterators retest on stg please.

https://d8-stg.boston.gov/departments/311/city-boston-government

**Comment by david-iterators** [comment]
Verified on Staging
